pyanaconda/ui/lib/storage.py

```diff
@@ -0,0 +1,303 @@
+#
+# Copyright (C) 2019  Red Hat, Inc.
```

Comment:
2020?

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2363.